libs/community/langchain_community/document_loaders/csv_loader.py

```diff
@@ -97,7 +97,9 @@ def __read_file(self, csvfile: TextIOWrapper) -> Iterator[Document]:
                     f"Source column '{self.source_column}' not found in CSV file."
                 )
             content = "\n".join(
-                f"{k.strip()}: {v.strip() if v is not None else v}"
+                f"""{k.strip() if k is not None else k}: {v.strip()
+                if v is isinstance(v, str) else ','.join(map(str.strip, v))
```

Comment:
just `if isinstance(v, str)` not `if v is isinstance(v, str)`

Reply:
Oh, I will correct it.

Reply:
@baskaryan I have corrected it, thank you for your help.